### PR TITLE
Feature/show session info in title

### LINKED ISSUE #15: Add the timer on the title of the page...
... so the user can easily see the current timer without switching to the app page.

[React Helmet](https://github.com/nfl/react-helmet) can be used to achieve this.

Think on a format to display it, maybe using the current entry type (pomodoro, short-break, long-break) and the timer, or just the timer, currently it's just showing `Tiempo`, examples:

Showing the type first and the lapse:
- `Pomodoro | 24:59`
- `Short Break | 4:56`

Also think in a good place to fit it, where does not affect the performance of the app.

### PR DESCRIPTION
**Pull request by jprask**
* Adds current session/lapse to title

* Ads tests for title

closes #15 

Before:

tiempo-tit-before

After: 

tiempo-tit-after

### CODE REVIEW

**Comment by jprask** [comment]
eslint is not happy about emoji a11y: `Emojis should be wrapped in <span>, have role="img", and have an accessible description with aria-label or aria-labelledby`

I think since they are in title it's not a concern... we could `// eslint-disable-next-line` it or use something like `25:00 | Pomodoro`

File: src/pomodoro/containers/PomodoroTitle.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import format from 'date-fns/format'
+import { Helmet } from 'react-helmet'
+import { connect } from 'react-redux'
+import * as fromPomodoro from 'pomodoro/store'
+
+const PomodoroTitle = ({ lapse, title }) => (
+  <Helmet>
+    <title>🕑 {format(lapse, 'mm:ss')} 🗒️ {title}</title>
```
Comment (iagodahlem) [inline src/pomodoro/containers/PomodoroTitle.js]: Very nice! I think we can just keep the "Tiempo" name on the end. Something like:

* `... | Tiempo`
* `... • Tiempo`
* `... - Tiempo`

Which one do you think is better?

File: src/pomodoro/containers/PomodoroTitle.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  </Helmet>
+)
+
+const mapStateToProps = state => ({
```
Comment (iagodahlem) [inline src/pomodoro/containers/PomodoroTitle.js]: Can you just move the `mapState` portion to below the propTypes declaration? Just to keep the consistency beyond the other components.

File: src/pomodoro/containers/PomodoroTitle.js (hunk 1 above)
Comment (iagodahlem) [inline src/pomodoro/containers/PomodoroTitle.js]: Now that we have the `format` function being used in two different places, what do you about creating a new component called `<Format />`, that receives a timestamp as children, and just return the result of the `format` function?

**Comment by iagodahlem** [comment]
I really liked the emojis on the title, I think we can try to use Unicode for the emojis instead of placing them directly, this we also show the emoji type for the system the user is using.

Not sure if it works, but I think it's worth the try. 

We can try to search for the emojis Unicode [here](http://www.amp-what.com/unicode/search/icon).

File: src/pomodoro/containers/PomodoroTitle.js (hunk 1 above)
Comment (jprask) [reply src/pomodoro/containers/PomodoroTitle.js]: Yeah, i think `... - Tiempo` looks better

**Pull request merged**